docs/Project.toml

```diff
@@ -1,5 +1,3 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
-
-[compat]
-Documenter = "~0.22"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
You can delete this unless you also committ manifest. It is beeing added here: https://github.com/JuliaArrays/StaticArrays.jl/blob/4b09eab5d5fbaba41495a61bf9bac603cc97a28a/.github/workflows/ci.yml#L62

Reply:
Thanks, I didn't know the relation.
I'll remove it.